src/NzbDrone.Common.Test/ExtensionTests/IPAddressExtensionsFixture.cs

```diff
@@ -21,6 +21,9 @@ public void should_return_true_for_local_ip_address(string ipAddress)
         [TestCase("1.2.3.4")]
         [TestCase("172.55.0.1")]
         [TestCase("192.55.0.1")]
+        [TestCase("100.64.0.1")]
+        [TestCase("100.127.255.254")]
```

Comment:
2 new tests for CGNAT IPs should be added, one to make sure they're not considered local and another for testing that they are CGNAT.